app/js/components/manageApps/Addon.jsx

```diff
@@ -73,11 +73,13 @@ class Addon extends Component {
     });
   }
 
-  handleAction(event, moduleUuid, action) {
-    event.preventDefault();
+  handleAction(moduleUuid, action, event=null) {
+    event? event.preventDefault(): this.hideModal();
```

Comment:
Curious about this implementation. If you do not mind, kindly take me through it.

Reply:
For the handleAction function it can be opened by a button event or opened from a modal. So that statement handles the process incase of an event or hides the modal from the display